gradle.properties

```diff
@@ -0,0 +1 @@
+systemProp.version = 2.1.0
```

Comment:
I assume the `version` upper/lower case does not matter here?

Reply:
Right.